beckend/database-models/index.ts

```diff
@@ -0,0 +1,18 @@
+import Sequelize from 'sequelize';
+
+const sequelize = new Sequelize("postgres://postgres:Vitalii19@localhost/Users", {
```

Comment:
Usually paths to db are stored in configuration file or ENV_VARIABLES, because in real project you have production db, dev db, qa db and you should have ability to easy switch between them.